cli/azd/pkg/project/muted_console.go

```diff
@@ -0,0 +1,76 @@
+package project
```

Comment:
Moved from being inline of `service_target_containerapp.go` since it isn't needed anymore in there.